activestorage/lib/active_storage/attached/model.rb

```diff
@@ -47,8 +47,17 @@ module Attached::Model
       #     has_one_attached :avatar, strict_loading: true
       #   end
       #
-      def has_one_attached(name, dependent: :purge_later, service: nil, strict_loading: false)
+      # If you need the attachment to have a specific storage path, on a Model level (other than the global 'route_prefix'),
+      # pass the +:key+ option. For instance:
+      #
+      #   class User < ActiveRecord::Base
+      #     has_one_attached :avatar, key: 'tenant/clients/4567/users/12322/avatars'
```

Comment:
This example shows a sample result instead of a sample config. Shouldn't we see some keys ":tenant" and ":user_id"?